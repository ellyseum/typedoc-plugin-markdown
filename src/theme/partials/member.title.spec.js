describe(`member.title.hbs`, () => {
  test('should compile', () => {
    const context = { name: 'member' };
    const result = global.compileTemplate('member.title.hbs', context);
    expect(result).toMatchSnapshot();
  });

  test('should compile with flags', () => {
    const context = { name: 'member', flags: ['Static'] };
    const result = global.compileTemplate('member.title.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
