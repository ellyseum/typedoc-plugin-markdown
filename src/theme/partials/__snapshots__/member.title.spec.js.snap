// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`member.title.hbs should compile 1`] = `"### member"`;

exports[`member.title.hbs should compile with flags 1`] = `"### \`<Static>\` member"`;
